docs/03-extensions/algolia/how-to/standalone.mdx

```diff
@@ -0,0 +1,171 @@
+---
+title: Standalone
+description: "This guides explains how to configure the Algolia extension to
+leverage custom indices."
+sidebar_position: 3
+---
+
+<p>{frontMatter.description}</p>
+
+## Front-Commerce
+
+[After installing the Algolia
+package](/docs/remixed/extensions/algolia/#installation), you need to enable the
+corresponding extension. For that, you need to tweak your
+`front-commerce.config.ts` file like:
+
+```typescript title="front-commerce.config.ts"
+import { defineConfig } from "@front-commerce/core/config";
+import themeChocolatine from "@front-commerce/theme-chocolatine";
+import magento2 from "@front-commerce/magento2";
+// highlight-next-line
+import algolia from "@front-commerce/algolia";
+import storesConfig from "./app/config/stores";
+import cacheConfig from "./app/config/caching";
+
+export default defineConfig({
+  extensions: [
+    // highlight-next-line
+    algolia("standalone"), // ⚠️ need to be before magento2() or magento1()
+    magento2(storesConfig), // or magento1()
+    themeChocolatine(),
+  ],
+  stores: storesConfig,
+  cache: cacheConfig,
+  configuration: {
+    providers: [],
+  },
+});
+```
+
+<a name="algolia-standalone-env"></a>
+In the .env, you need to define the following environment variables:
+
+```bash title=".env"
+FRONT_COMMERCE_ALGOLIA_APPLICATION_ID=APP_ID
+FRONT_COMMERCE_ALGOLIA_SEARCH_ONLY_API_KEY=SEARCH_ONLY_API_KEY
+FRONT_COMMERCE_ALGOLIA_INDEX_NAME_PREFIX=myproject_
+```
+
+<a name="algolia-standalone-config"></a>
+
+<!--
+TODO: this feature does not existing yet in V3
```

Comment:
wrong assumption.
self note: uncomment and point to a WIP page about configuration provider